docs/pbc/all/search/202410.0/base-shop/tutorials-and-howtos/configure-elasticsearch.md

```diff
@@ -187,3 +187,16 @@ For the default page index, the class is `\Generated\Shared\Search\PageIndexMap`
 These classes provide some information from mapping, such as fields and metadata. Use these classes for references to program against something related to that mapping schema.
 
 If you change mapping and run the installer, autogenerated classes change accordingly.
+
+{% info_block warningBox "Index update limitations" %}
+
+Elastic Search has different limitations, when it comes to update of the existing index with data in it.
+Unfortunatelly, the errors you see from the Elastic Search are confusing.
+In order to ensure that the index is correct, please drop the index, create it again and sync data into it.
+APPLICATION_STORE=DE console search:index:delete
+APPLICATION_STORE=DE console search:setup:sources
+APPLICATION_STORE=DE console sync:data
```

Comment:
worth to mention alternative `console publish:trigger-events`

Reply:
Well, my description is not mentioning this part at all, see line 195.

Reply:
Just checked your changes ) So all is correct